SourcetrailExtensionTests/IntegrationTests/CreateCdbTests.cs

```diff
@@ -67,6 +67,17 @@ public void TestCompilationDatabaseCreationForAllFilesInSameFolder()
 
 		[TestMethod]
 		[HostType("VS IDE")]
+		public void TestCompilationDatabaseCreationForCinderSolution_NonSystemIncludes()
```

Comment:
New test, but the only one with the now-default behavior (just to avoid churn in the test data).